jgrapht-io/src/main/java/org/jgrapht/nio/tsplib/TSPLIBImporter.java

```diff
@@ -619,9 +618,11 @@ public void importGraph(Graph<V, E> graph, Reader in)
         return nodes;
     }
 
+    private static final Pattern WHITE_SPACE = Pattern.compile("[ \t]+");
```

Comment:
The pattern "[ \t]+" allows an arbitrary number (greater zero) of spaces and tabs to be used as delimiter.

Instead of " \t" the predefined character class "\\s+" could be used to allow arbitrary white-spaces.
From the java-doc of the `java.util.regex.Pattern` class:
```
Predefined character classes
\s  |  A whitespace character: [ \t\n\x0B\f\r]
```

I think it is very unlikely that arbitrary white-spaces are used (especially since \n and \r are used as line-separator).
What do you think?

Reply:
Looks fine to me. Unlikely that anything else than tabs or spaces will be used.